strings/loot_data/all_spawn_data.txt

```diff
@@ -2013,6 +2013,7 @@ paths    spawn_tags    blacklisted    spawn_value    price_tag    prob_all_accom
 /obj/item/weapon/gun_upgrade/mechanism/overshooter    obj,item,gun_upgrade    blacklisted=0    spawn_value=5    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow    obj,item,gun_upgrade    blacklisted=1    spawn_value=2    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/weintraub    obj,item,gun_upgrade    blacklisted=0    spawn_value=3.33333    price_tag=100   100    
+/obj/item/weapon/gun_upgrade/mechanism/funswitch    obj,item,gun_upgrade    blacklisted=0    spawn_value=3.33333    price_tag=100   100    
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.

Reply:
Should hopefully be removed from the PR.